lib/server.js

```diff
@@ -24,6 +24,17 @@ var parseJobLink = function(request, respond) {
   return decodeURIComponent(jobLink);
 };
 
+function slackUrlToJenkins(url) {
+  var fragments = decodeURIComponent(url).
+    match(/[\/?&]text=([^&]+)/)[1].
```

Comment:
Why do you need to match `/` in before the query parameter? Shouldn't it be either `?text` or `&text`?